src/components/modal/AddNewBikeServiceModal.js

```diff
@@ -0,0 +1,91 @@
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+import { useState } from "react";
+import { isEmpty } from "../../stringHelper";
+import { addBikeServiceRequest } from "../../services/bikeServiceService";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+function AddNewBikeServiceModal(props) {
+  const [name, setName] = useState("");
+  const [value, setValue] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  function handleValueChange(event) {
+    setValue(event.target.value);
+  }
+
+  function handleNameChange(event) {
+    setName(event.target.value);
+  }
+
+  function isValidEntrances() {
+    return !isEmpty(name) && !isEmpty(value);
+  }
+
+  function createBikeService() {
+    if (isValidEntrances()) {
+      addBikeServiceRequest(name, value)
+        .then((_) => {
+          toast.success("Serviço criado com sucesso!", {
+            position: "top-right",
+            autoClose: false,
```

Comment:
Se a config for igual para todos, pode criar uma constante em algum lugar e reaproveitar